libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,375 @@
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "rockfish",
+          "Name": "Rockfish",
+          "PartyId": "1"
+        },
+        {
+          "@type": "CVR.Candidate",
+          "@id": "triggerfish",
+          "Name": "Triggerfish",
+          "PartyId": "1"
+        }
+      ],
+      "Contest": [
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "best-animal-fish",
+          "Name": "Best Animal",
+          "NumberElected": 1,
+          "VotesAllowed": 1,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["seahorse"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["salmon"]
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.CandidateContest",
+          "@id": "aquarium-council-fish",
+          "Name": "Zoo Council",
+          "NumberElected": 2,
+          "VotesAllowed": 2,
+          "ContestSelection": [
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["manta-ray"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["pufferfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["rockfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "CandidateIds": ["triggerfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "isWriteIn": true
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "isWriteIn": true
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.BallotMeasureContest",
+          "@id": "fishing",
+          "Name": "Ballot Measure 3",
+          "ContestSelection": [
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "yes",
+              "Selection": "yes"
+            },
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "no",
+              "Selection": "no"
+            }
+          ],
+          "PrimaryPartyId": "1"
+        }
+      ],
+      "ElectionScopeId": ["sample-county"]
+    }
+  ],
+  "GpUnit": [
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "sample-county",
+      "Name": "Sample County",
+      "Type": "other"
+    },
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "precinct-1",
+      "Name": "Precinct 1",
+      "Type": "precinct"
+    }
+  ],
+  "CVR": [
+    {
+      "@type": "CVR.CVR",
+      "BallotStyleId": "2F",
+      "BallotStyleUnitId": "precinct-1",
+      "PartyIds": ["1"],
+      "CreatingDeviceId": "scanner-1",
+      "ElectionId": "some-id",
+      "UniqueId": "id-168",
```

Comment:
The CDF description of this field is:

> The sequence number for this CVR. This represents the ordinal number that this CVR was processed by the tabulating device

Our actual IDs (not in these fixtures) are not serial, however, they're UUIDs. Although we'd be going outside of the description, I think what we have is a "Unique ID" and we should just use this field.

Reply:
Sequence number is useful in RLAs. Are the UUIDs ordered based on tabulation order? If so, then that's fine. If not, then I might replace this with an incrementing counter of some sort.

Reply:
Oh wait, but I see there's `BatchSequenceId` for that... Very confusing

Reply:
Yes, I think there are some issues with the documentation around here, a couple descriptions that look more copy pasted than written.

Reply:
I think on the precinct scanner we don't _want_ a sequence ID because that would compromise voter privacy cc @mattroe. On central scanner i'll probably have a follow-up task to make it work.

Reply:
Yeah, my perspective was sequence ID on central scanned batches, but no ID on precinct scanned batches for that reason unless someone is thinking differently. VVSG says "if applicable" so I assume that is to allow to not include it where it would be a privacy violation.